systemd/gssproxy.service.in

```diff
@@ -6,6 +6,7 @@ Before=rpc-gssd.service
 [Service]
 ConfigurationDirectory=gssproxy
 StateDirectory=gssproxy gssproxy/clients gssproxy/rcache
+StateDirectoryMode=0700
```

Comment:
Why this change ?

Reply:
Ok, I do not think there is any problem leaving this as 755 even in the spec file, given files are always created with restrictive permissions, but I do not see a huge deal in making it root only either ... it just makes the "run_as_user" setting difficult to use, because now you have to run around and change a bunch of directory permissions I guess ... but I never cared too much for tat setting.

Reply:
Since systemd overrides the cache directories according to the service unit file content, I guess that if the `User` setting is modified, it will set the owner of the cache directories accordingly. So it should not get in the way of non-root users to run the daemon.

The main motivation is to silent the rpminspect tests. Because gssproxy is started as part of the post-install script it modifies the directories mode immediately, which is causing installation tests to fail decause mode does not match the specfile anymore.

Reply:
yeah, that is fine.